include/z64.h

```diff
@@ -615,227 +624,257 @@ typedef struct {
+    /* 0x1A0 */ Vtx* unk_1A0;
+    /* 0x1A4 */ Vtx* unk_1A4;
+    /* 0x1A8 */ Vtx* cursorVtx;
+    /* 0x1AC */ UnkAudioStruct* unk_1AC;
+    /* 0x1B0 */ DmaRequest unk_1B0;
+    /* 0x1D0 */ OSMesgQueue loadQueue;
+    /* 0x1E8 */ OSMesg loadMsg;
+    /* 0x1EC */ u16 state;
+    /* 0x1EE */ u16 debugState;
+    /* 0x1F0 */ u8 unk_1F0;
+    /* 0x1F4 */ Vec3f eye;
+    /* 0x200 */ u16 unk_200;
+    /* 0x202 */ u16 mode;
+    /* 0x204 */ u16 pageIndex;
+    /* 0x206 */ u16 unk_206;
+    /* 0x208 */ u16 unk_208;
+    /* 0x20C */ f32 unk_20C;
+    /* 0x210 */ f32 unk_210;
+    /* 0x214 */ f32 unk_214;
+    /* 0x218 */ f32 unk_218;
+    /* 0x21C */ f32 unk_21C;
+    /* 0x220 */ f32 unk_220;
+    /* 0x224 */ u16 alpha;
+    /* 0x226 */ s16 offsetY;
+    /* 0x228 */ s32 unk_228;
+    /* 0x22C */ s32 unk_22C;
+    /* 0x230 */ s32 unk_230;
+    /* 0x234 */ s32 unk_234;
+    /* 0x238 */ s16 unk_238[5];
+    /* 0x242 */ s16 unk_242[5];
+    /* 0x24C */ s16 unk_24C[5];
+    /* 0x256 */ s16 unk_256;
+    /* 0x258 */ s16 unk_258;
+    /* 0x25A */ s16 unk_25A;
+    /* 0x25C */ u16 unk_25C;
+    /* 0x25E */ u16 unk_25E[5];
+    /* 0x268 */ u16 unk_268[5];
+    /* 0x272 */ u16 equipTargetItem;
+    /* 0x274 */ u16 equipTargetSlot;
+    /* 0x276 */ u16 equipTargetCBtn;
+    /* 0x278 */ s16 equipAnimX;
+    /* 0x27A */ s16 equipAnimY;
+    /* 0x27C */ s16 equipAnimAlpha;
+    /* 0x27E */ s16 unk_27E;
+    /* 0x280 */ u16 unk_280;
+    /* 0x282 */ u16 unk_282;
+    /* 0x284 */ s16 unk_284;
+    /* 0x286 */ s16 unk_286;
+    /* 0x288 */ f32 unk_288;
+    /* 0x28C */ f32 unk_28C;
+    /* 0x290 */ f32 unk_290;
+    /* 0x294 */ f32 unk_294;
+    /* 0x298 */ f32 unk_298;
+    /* 0x29C */ s16 unk_29C;
+    /* 0x29E */ s16 unk_29E;
+    /* 0x2A0 */ s16 unk_2A0;
+    /* 0x2A2 */ u8 worldMapPoints[20];
+    /* 0x2B6 */ u8 unk_2B6;
+    /* 0x2B7 */ u8 unk_2B7;
+    /* 0x2B8 */ u8 unk_2B8;
+    /* 0x2B9 */ u8 unk_2B9;
+    /* 0x2BA */ s16 unk_2BA;
+    /* 0x2BC */ s16 unk_2BC;
+    /* 0x2BE */ s16 unk_2BE;
+    /* 0x2C8 */ u16 unk_2C8;
+    /* 0x2CA */ s16 unk_2CA;
 } PauseContext; // size = 0x2D0
 
 typedef struct {
     /* 0x000 */ View view;
-    /* 0x168 */ char unk_168[0xB4];
+    /* 0x168 */ Vtx* actionVtx;
+    /* 0x16C */ Vtx* beatingHeartVtx;
+    /* 0x170 */ u8* parameterSegment;
+    /* 0x174 */ u8* doActionSegment;
+    /* 0x178 */ u8* iconItemSegment;
+    /* 0x17C */ u8* mapSegment;
+    /* 0x180 */ u8* unk_180;
+    /* 0x184 */ DmaRequest dmaRequest_184;
+    /* 0x1A4 */ DmaRequest dmaRequest_1A4;
+    /* 0x1C4 */ DmaRequest dmaRequest_1C4;
+    /* 0x1E4 */ OSMesgQueue loadQueue;
+    /* 0x1FC */ OSMesg loadMsg;
+    /* 0x200 */ Viewport viewport;
+    /* 0x210 */ s16 unk_210;
+    /* 0x212 */ u16 unk_212;
+    /* 0x214 */ u16 unk_214;
+    /* 0x218 */ f32 unk_218;
     /* 0x21C */ s16 unk_21C;
     /* 0x21E */ s16 unk_21E;
-    /* 0x220 */ char unk_220[6];
-    /* 0x226 */ s16 lifeColorChange;
-    /* 0x228 */ s16 lifeColorChangeDirection;
```

Comment:
I notice you are reverting some named fields to unk fields. Can you justify why this is done? I'm not sure it's beneficial to revert names to unk unless they're totally wrong.

Reply:
This one I meant to add back but I remember this not exactly being what is described here in oot and I wanted to look into it more.